app/models/preprint.js

```diff
@@ -1,17 +1,37 @@
 import DS from 'ember-data';
+import OsfModel from 'ember-osf/models/osf-model';
 
-export default DS.Model.extend({
+/**
+ * @module ember-preprints
+ * @submodule models
+ */
+
+/**
+ * Model for OSF APIv2 preprints. This model may be used with one of several API endpoints. It may be queried directly,
+ *  or accessed via relationship fields.
+ * For field and usage information, see:
+ * https://api.osf.io/v2/docs/#!/v2/Preprint_List_GET
+ * https://api.osf.io/v2/docs/#!/v2/Preprint_Detail_GET
+ * https://api.osf.io/v2/docs/#!/v2/User_Preprints_GET
+ * @class Preprint
+ */
+export default OsfModel.extend({
     title: DS.attr('string'),
-    authors: DS.attr(),
-    date: DS.attr(),
-    subject: DS.attr(),
-    abstract: DS.attr(),
-    publisher: DS.attr(),
-    project: DS.attr(),
-    supplementalMaterials: DS.attr(),
-    figures: DS.attr(),
-    license: DS.attr(),
-    path: DS.attr(),
+    // TODO: May be a relationship in the future pending APIv2 changes
+    subjects: DS.attr(),
```

Comment:
My understanding is that the taxonomy endpoint is a giant bucket of every possible subject name at any level of a nested hierarchy.  The "subjects" field reflects the child categories of that given taxonomy category.

However, this isn't currently reflected in the test data, so pinging @erinspace to verify.